client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -57,7 +57,7 @@ import { activateModule } from 'state/jetpack/modules/actions';
 import { isBusiness, isEnterprise, isJetpackBusiness } from 'lib/products-values';
 import { hasFeature } from 'state/sites/plans/selectors';
 import { getPlugins } from 'state/plugins/installed/selectors';
-import { FEATURE_ADVANCED_SEO, PLAN_BUSINESS } from 'lib/plans/constants';
+import { FEATURE_SEO_PREVIEW_TOOLS, PLAN_BUSINESS } from 'lib/plans/constants';
```

Comment:
I understand that we fail to display the banner for Jetpack sites, however by updating this one to appear for Jetpack sites, it will no longer appear for .com sites. This is because the Jetpack Professional plan contains has the `FEATURE_SEO_PREVIEW_TOOLS` feature, while the .com Business plan contains the `FEATURE_ADVANCED_SEO` feature.

This is a regression IMHO. We need to make sure that if we make this banner appear for Jetpack sites, it continues to appear for .com sites as well.

Reply:
Great catch 👍 
Let's fix it 😄 
https://github.com/Automattic/wp-calypso/pull/20263